corehq/apps/reports/views.py

```diff
@@ -162,6 +164,24 @@
 require_can_view_all_reports = require_permission(HqPermissions.view_reports)
 
 
+def _augmented_form_view_permissions(login_decorator=login_and_domain_required):
+    def decorator(view_func):
+        @wraps(view_func)
+        def _inner(request, domain, *args, **kwargs):
+            if VIEW_FORM_ATTACHMENT.enabled(domain):
+                return view_func(request, domain, *args, **kwargs)
+            return require_form_view_permission(view_func)(request, domain, *args, **kwargs)
+
+        if login_decorator:
+            return login_decorator(_inner)
+        else:
+            return _inner
+    return decorator
+
+
+augmented_form_view_permissions = _augmented_form_view_permissions(login_decorator=None)
```

Comment:
why does this set `login_decorator` to `None`?  That means this view won't be protected

Reply:
The place where the `augmented_form_view_permissions` decorator is used is already using api_auth (see [here](https://github.com/dimagi/commcare-hq/blob/1bc480bc704ac412c2725cf3c992c00e6192802c/corehq/apps/api/object_fetch_api.py#L131)).

I've basically copied the `require_form_view_permission` behaviour as seen [here](https://github.com/dimagi/commcare-hq/blob/cabd9fe44b06ad753f160f763c4704f89a76a5f9/corehq/apps/reports/views.py#L161).